.github/workflows/ci.yaml

```diff
@@ -147,16 +147,26 @@ jobs:
       REF_PRAXIS: ${{ needs.metadata.outputs.REF_PRAXIS }}
     secrets: inherit
 
-  build-rosetta:
+  build-rosetta-t5x:
     uses: ./.github/workflows/_build_rosetta.yaml
     needs: [metadata, build-t5x]
     with:
       BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
       BASE_IMAGE: ${{ needs.build-t5x.outputs.DOCKER_TAGS }}
+      BASE_LIBRARY: t5x
+    secrets: inherit
+
+  build-rosetta-pax:
+    uses: ./.github/workflows/_build_rosetta.yaml
+    needs: [metadata, build-pax]
+    with:
+      BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
+      BASE_IMAGE: ${{ needs.build-pax.outputs.DOCKER_TAGS }}
+      BASE_LIBRARY: pax
```

Comment:
It looks like this will trigger the exact error that I hypothesized above :grin: